public/assets/icons/add.tsx

```diff
@@ -1,7 +1,7 @@
 const AddIcon = () => (
     <svg width="30px" height="30px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg">
         <title>Create new</title>
-        <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Page-1" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
```

Comment:
why is this change required....any specific reason?
I think previous code also looked good

Reply:
This is a syntax convention.
![image](https://github.com/Real-Dev-Squad/tiny-site-frontend/assets/53934353/0f659a5c-4f5c-458e-a48c-34d4e856317f)